src/drawer.js

```diff
@@ -500,6 +500,10 @@ $.Drawer.prototype = {
 
         if ( this.viewport.degrees !== 0 ) {
             this._offsetForRotation({degrees: this.viewport.degrees});
+        } else{
+          if(this.viewer.flipped) {
+            this._flip({});
```

Comment:
When the options are optional, we often add `options = options || {}` at the top of their function so we don't have to pass `{}` into the function. Let's do that in this case too.